src/System/Taffybar/ToggleMonitor.hs

```diff
@@ -0,0 +1,73 @@
+{-# LANGUAGE OverloadedStrings #-}
+module System.Taffybar.ToggleMonitor (
+  handleToggleRequests,
+  toggleableMonitors,
+  withToggleServer
+) where
+
+import           Control.Concurrent
+import qualified Control.Concurrent.MVar as MV
+import           Control.Monad
+import           Control.Monad.Trans
+import           Control.Monad.Trans.Maybe
+import qualified Data.Map as M
+import           Data.Maybe
+import qualified Graphics.UI.Gtk as Gtk
+import           Graphics.UI.Gtk.Gdk.Screen
+import           System.Taffybar
+import           Text.Read hiding (get, lift)
+import           Web.Scotty
```

Comment:
I have to say I'm a bit reluctant to bring an entire web server into the process.  Is there a reason to prefer something that heavy-weight?  We already have a d-bus dependency, which could serve much the same purpose.  A file interface might be even simpler.